examples/alb-test/tester.tf

```diff
@@ -0,0 +1,64 @@
+module "vpc" {
+  source              = "../../modules/vpc-scenario-2"
+  cidr                = "192.168.0.0/16"
+  region              = "us-east-2"
+  name_prefix         = "alb_test"
+  public_subnet_cidrs = ["192.168.1.0/24", "192.168.2.0/24", "192.168.3.0/24"]
+  azs                 = ["us-east-2a", "us-east-2b", "us-east-2c"]
+}
+
+module "asg" {
+  source               = "../../modules/asg"
+  ami                  = "ami-0efd7a9e366418dda"
+  key_name             = "shida-east-2"
+  azs                  = ["us-east-2a", "us-east-2b", "us-east-2c"]
+  max_nodes            = 3
+  min_nodes            = 3
+  subnet_ids           = module.vpc.public_subnet_ids
+  name_prefix          = "alb_test"
+  security_group_ids   = [aws_security_group.wide_open.id]
+  lb_target_group_arns = module.alb.target_group_arns
+}
+
+resource "aws_security_group" "wide_open" {
+  vpc_id = module.vpc.vpc_id
+
+  ingress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+}
+
+module "alb" {
+  source        = "../../modules/alb"
+  ports         = ["80"]
+  vpc_id        = module.vpc.vpc_id
+  sg_ids        = [aws_security_group.wide_open.id]
+  identifier    = "albest"
+  subnet_ids    = module.vpc.public_subnet_ids
+  ports_secured = []
+  dns_name      = "shida.dev-sandbox.fpcomplete.com"
```

Comment:
@Magicloud do we need to hardcode your name here and for the key name above?

Reply:
It does not. I'd like this to be a unit test and planable without human interfere, so the code can be more robust.. But that would require a framework, that provides necessary variables in CI. Nothing done on that yet.

Reply:
If we don't have a good way resolve this then probably my comment could be ignored...